lib/ansible/plugins/filter/core.py

```diff
@@ -65,6 +65,13 @@ def to_nice_yaml(a, indent=4, *args, **kw):
 
 def to_json(a, *args, **kw):
     ''' Convert the value to JSON '''
+
+    # defualts for filters
+    if 'vault_to_text' not in kw:
+        kw['vault_to_text'] = True
+    if 'preprocess_unsafe' not in kw:
+        kw['preprocess_unsafe'] = False
```

Comment:
I purposefully didn't do this to not break backwards compat.  I've been recommending that people pass these kwargs explicitly if that wanted that functionality.

Reply:
this actually restores the backward compatible behavior, while still allowing users to be specific